changes/1119.breaking.md

```diff
@@ -0,0 +1 @@
+Return `LazyIterator[GuildBan]` from `RESTClientImpl.fetch_bans` instead of `Sequence[GuildBan]`.
```

Comment:
This really doesnt give any information to unexperienced users.

Something along these lines instead?
```suggestion
`RESTClientImpl.fetch_bans` now return a `LazyIterator` to allow pagination of values.
```

Also, this isnt really breaking as awaiting the iterator will just cause the same functionality as before. I would probably label this as a bugfix.